### DISCUSSION TITLE
slack notification - deployment status

### DISCUSSION THREAD

**Discussion by jazanne** [community/community#9137] [Actions]
In the GitHub UI, when a workflow is cancelled it looks like this:

![CleanShot 2021-12-20 at 09 40 42](https://user-images.githubusercontent.com/4053924/146784660-3e656f8e-0290-4bbd-8225-e792c9d8396c.png)

In slack - when it's a deployment workflow that is cancelled it shows Status as "Failed", which is very alarming!

![CleanShot 2021-12-20 at 09 42 06@2x](https://user-images.githubusercontent.com/4053924/146784890-567a67dd-6301-46d1-b61a-400c048390d1.png)

It would be great is deployment workflows that are cancelled set Status as "Cancelled", so that our team doesn't launch an investigation into failed deployments unnecessarily

**Comment by ethomson** [comment]
Thanks for the feedback @jazanne - this makes a lot of sense.  I've passed this on to the slack app team.

**Reply by jazanne** [reply]
awesome - thank you!

**Reply by robdmoore** [reply]
This is still a problem, was there any reply from that team about this?
